files/build_templates/sonic_debian_extension.j2

```diff
@@ -1193,6 +1192,20 @@ sudo rm -rf $FILESYSTEM_ROOT/tmp/mask_disabled_services.py
 
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install python3-dbus
 
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get purge -y python3-dev
```

Comment:
>purge -y python3-dev [](http://example.com/codeflow?start=75&length=20)

Could you move this new code block to original place of purging python3-dev?

Reply:
Fixed